browser/css/jssidebar.css

```diff
@@ -130,12 +130,12 @@ div.sidebar.ui-grid .checkbutton.sidebar,
 	position: absolute;
 	display: block;
 	width: auto;
-	background-color: var(--color-background-darker);
+	background-color: transparent;
```

Comment:
I think we should make sure the hover status for the menubutton has the same border color and background color as other components such as.jsdialog.ui-listbox:hover 

also `.menubutton.sidebar:hover span` should be improved  ideally to look like any other regular `.ui-tooltip`